docs/software/spack.md

```diff
@@ -143,7 +143,7 @@ In addition to standard software packages, you can use Spack to install compiler
     spack install --add gcc@13.1.0
```

Comment:
The beginning statement of this page is incorrect, Spack is now available on Alpine and Blanca. Please correct that.

Reply:
I am still seeing "not currently be available in the Alpine software stack."

Reply:
I do see you addressed the beginning part.